src/Command/ReleaseCommand.php

```diff
@@ -92,7 +111,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $this->io->getErrorStyle()->title($project->name());
 
-        return $this->renderNextRelease($project, $branch);
+        $nextRelease = $this->renderNextRelease($project, $branch);
+
+        if (null !== $nextRelease && $this->createPR($input, $output)) {
+            $this->prepareReleasePR($nextRelease);
+        }
+
+        return null !== $nextRelease ? 0 : 1;
```

Comment:
We can use self::Success or self::Fail constant, I think something like this exist in Command instead of 0 or 1.

Reply:
Fixed.